docs/source/conf.py.in

```diff
@@ -80,16 +73,15 @@
 source_suffix = '.rst'
 
 # The master toctree document.
-master_doc = 'intro'
+master_doc = 'index'
 
 # General information about the project.
 project = 'Rascal'
-copyright = ('2019, Chiheb Ben Mahmoud, Michele Ceriotti, Federico Giberti, ' +
-             'Klim Goldshtein, Till Junge, Markus Stricker, Félix Musil, ' +
-             'Max Veit')
-author = ('Michele Ceriotti, Federico Giberti, Alexander Gocsinski, ' +
-          'Till Junge, Félix Musil, Markus Stricker, Max Veit, and ' +
-          'Michael Willatt')
+copyright = '2019, Chiheb Ben Mahmoud, Michele Ceriotti, Federico Giberti, \
```

Comment:
Ergh, backslash line continuation. IIRC this has been basically deprecated in Python for a while, except where it's unavoidable; to cite PEP8: 
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Reply:
Ok, I am more used to C++ here. Should this use a multiline string ? `"""..."""`